scripts/install-service.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e 
+
+# Define data directory to be used for application config file.
+DATA_DIR=${DATA_DIR:-${HOME}/.poglink}
+
+# Change to script directory for predictable behaviour
+cd "$(dirname "$0")"
+
+# Check whether virtual environment is active
+if [ -z "${VIRTUAL_ENV}" ] && [ "${FORCE}" != "true"  ]; then
```

Comment:
If `FORCE=true` then this block will pass and it will install `poglink` on the system Python environment. This might be desireable if using a dedicated VM for this bot and didn't want to bother with a `venv` when you knew there was nothing to worry about conflicting with.